### PR TITLE
#338: Add Quickstart 2.0 indicator to toolbar.

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Toolbar quickstart version indicator implementation created during local development workflow demo in 2020-11-04 workshop.

## Related Issue
#338 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart project in the right column.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by camikazegreen** [comment]
I think that we should either:
- change this to use a variable that will automatically keep up with minor version updates. Example: 2.1
- change this to just reference "Quickstart 2" so that it isn't implying a more specific version number

**Comment by joeparsons** [comment]
TODO:
- More (or less) specific version as suggested by @camikazegreen 
- Add more specific class and stylesheet to float right

File: modules/custom/az_core/az_core.module [hunk 1]
```diff
@@ -24,6 +25,31 @@ function az_core_help($route_name, RouteMatchInterface $route_match) {
...
+ * Implements hook_toolbar().
+ */
+function az_core_toolbar() {
+  $items = [];
+
+  $items['az_quickstart'] = [
+    'type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'link',
+      '#title' => t('Quickstart 2.0'),
```
Comment (camikazegreen) [inline modules/custom/az_core/az_core.module:38]:
```suggestion
      '#title' => t('Quickstart 2'),
```

File: modules/custom/az_core/az_core.module [hunk 2]
```diff
@@ -24,6 +25,31 @@ function az_core_help($route_name, RouteMatchInterface $route_match) {
...
+      '#url' => Url::fromRoute('az_core.az_quickstart'),
+      '#attributes' => [
+        'title' => t('AZ Quickstart 2.0'),
```
Comment (camikazegreen) [inline modules/custom/az_core/az_core.module:41]:
```suggestion
        'title' => t('AZ Quickstart 2'),
```

Comment (camikazegreen) [reply modules/custom/az_core/az_core.module:41]: I think that making this change is enough to satisfy this ticket and make a good addition to Quickstart without spending a lot of additional effort on a low priority feature.

**Review by camikazegreen — DISMISSED** [review]

**Comment by camikazegreen** [comment]
@joeparsons Does this need to have _main_ pulled in to resolve the security steps before we merge?

**Review by camikazegreen — APPROVED** [review]

**Review by smayden — APPROVED** [review]
This looks good to me from a content editor/admin UI stand point.

**Review by trackleft — APPROVED** [review]

**Pull request merged**